src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -509,6 +576,21 @@ export default class rd2EntryForm extends LightningElement {
      */
     handleSaveError(error) {
         this.error = constructErrorMessage(error);
+
+        if (isNull(this.recordId) && !isEmpty(this.getCommitmentId())) {
+            const message = 'A recurring commitment has been created in Elevate with tracking "Elevate Recurring Id" {0}. ' //TODO custom label
```

Comment:
@sam-knox , @mattwashington : this is a temp error message displayed to a user when the commitment in Elevate is created but the Recurring Donation insert fails. User can fix the issue and save again OR wait for CCS integration to attempt creation again (this is async process). This error message is related to the [W-8554656](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008l8I5IAI/view).

Reply:
Thanks @RadSFDeveloper, I can take this one.